sdk/healthinsights/azure-healthinsights-radiologyinsights/azure/healthinsights/radiologyinsights/_operations/_operations.py

```diff
@@ -141,122 +156,128 @@ def _infer_radiology_insights_initial(
-                                        "meta": {
-                                            "lastUpdated": "str",  #
-                                              Optional. When the resource last changed - e.g. when the
-                                              version changed.
-                                            "profile": [
-                                                "str"  # Optional. A
-                                                  list of profiles (references to `StructureDefinition
-                                                  <https://www.hl7.org/fhir/structuredefinition.html>`_
-                                                  resources) that this resource claims to conform to.
-                                                  The URL is a reference to `StructureDefinition.url
-                                                  <https://www.hl7.org/fhir/structuredefinition-definitions.html#StructureDefinition.url>`_.
-                                            ],
-                                            "security": [
-                                                {
-                                                    "code":
-                                                      "str",  # Optional. Symbol in syntax defined by
-                                                      the system.
-                                                    "display":
-                                                      "str",  # Optional. Representation defined by the
-                                                      system.
-                                                    "extension":
-                                                      [
-                                                        {
-                "url": "str",  # Source of the definition
-                                                              for the extension code - a logical name
-                                                              or a URL. Required.
-                "valueBoolean": bool,  # Optional. Value
-                                                              as boolean.
+                resource = {
+                    "id": "str",  # The unique ID of the job. Required.
+                    "status": "str",  # The status of the job. Required. Known values are:
+                      "notStarted", "running", "succeeded", "failed", and "canceled".
+                    "createdAt": "2020-02-20 00:00:00",  # Optional. The date and time when the
+                      processing job was created.
+                    "error": {
+                        "code": "str",  # One of a server-defined set of error codes.
+                          Required.
+                        "message": "str",  # A human-readable representation of the error.
+                          Required.
+                        "details": [
+                            ...
+                        ],
```

Comment:
Just highlighting this section as an example but the docstring is much longer than what I'm highlighting. I dont think these docstrings will be very readable for customers using intellisense on their code. Since it seems these docstrings are coming from the tsp, not blocking this PR on this, but I would make an issue to improve the documentation so that it's not as large.